app/src/main/kotlin/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -221,15 +217,13 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
     private var videoHeight = 0
     private var firstAuthTime: Long = 0
     private var resourceURL: String = ""
-    private var appLocked = true
+    private var appLocked = mutableStateOf(true)
     private var unlockingApp = false
-    private var exoPlayer: ExoPlayer? = null
+    private var exoPlayer = mutableStateOf<ExoPlayer?>(null)
     private var isExoFullScreen = false
-    private var exoTop = 0 // These margins are from the DOM and scaled to screen
-    private var exoLeft = 0
-    private var exoRight = 0
-    private var exoBottom = 0
-    private var exoMute = true
+    private var playerSize = mutableStateOf<DpSize?>(null)
+    private var playerTop = mutableStateOf<Dp>(0.dp)
+    private var playerLeft = mutableStateOf<Dp>(0.dp)
```

Comment:
The type can be inferred so isn't needed

```suggestion
    private var playerTop = mutableStateOf(0.dp)
    private var playerLeft = mutableStateOf(0.dp)
```

(after making this change, the import statement is unused)